src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -2960,6 +2960,171 @@ mesh::topology::unstructured::generate_sides(const Node &topo,
     blueprint::o2mrelation::generate_offsets(d2smap, info);
 }
 
+//-----------------------------------------------------------------------------
+template<typename T, typename U> 
+void 
+map_field(Node &field_out, 
```

Comment:
These internal funcs should also be `map_fields_to_generated_sides` or some derivative